scripts/install.sh

```diff
@@ -225,19 +225,23 @@ kustomize edit set namespace "$namespace"
 cd ..
 
 # Clean up previous instances but do not fail if they do not exist
+cp dependencies/quickstart.yaml .
+sed -i.bak "s/etcd:2379/etcd.${namespace}:2379/g" quickstart.yaml
```

Comment:
you can use `sed -pi -e` instead of generating the bak file.

Reply:
sed is implemented differently in OSX and Linux (GNU). According to the [spec](https://pubs.opengroup.org/onlinepubs/9699919799/), -p is not standard. `sed -pi -e` actually fails in Ubuntu/GNU sed. The use of bak works for both GNU and BSD versions of sed.